articles/urls.py

```diff
@@ -3,7 +3,9 @@
 from . import views
 
 router = DefaultRouter()
-router.register('', views.ArticleViewSet, 'articles')
+router.register('article', views.ArticleViewSet)
```

Comment:
`Router.register()` needs a third argument (the name of the endpoint) in order for `reverse()` to work properly.
```suggestion
router.register('', views.ArticleViewSet, 'articles')
```